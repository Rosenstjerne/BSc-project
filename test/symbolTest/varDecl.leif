var int x, y;
x = 10;
y = 20;
print(x + y); # Should print 30