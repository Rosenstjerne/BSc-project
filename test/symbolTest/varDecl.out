function null main() {
	var int x, y;
	
	x = 10;
	y = 20;
	print((x + y));
	
}
